### DISCUSSION TITLE
Dream Capture

### DISCUSSION THREAD

**Discussion by cnu1812** [mattermost/community#84] [Moonshot Ideas]
Seeing the title you may think it weird..!

   There is a part in the brain called Hippocampus where our memories/dreams are stored. Hippocampus. The hippocampus, located in the brain's temporal lobe, is where episodic memories are formed and indexed for later access. Episodic memories are autobiographical memories from specific events in our lives, like the coffee we had with a friend last week.

What we have to do is..rendering the memories from the hippocampus to our machine(desktop, mobile..).
This title can also be renamed as Time machine(which can travel to the past! I don't think we can make a time machine that travels to the future. But in coming years we can make a time machine which can travel to the past..!

**Comment by chetanyakan** [comment]
Another interesting idea would be to be able to record movies or video clips out of our memories/dreams and relive those moments whenever.

**Reply by cnu1812** [reply]
Great!
